public/pages/editor/editor.controller.js

```diff
@@ -63,6 +63,14 @@ function EditorController(sentinlConfig, $rootScope, $scope, $route, $interval,
       },
       rawEnabled: false,
     };
+    if (editorMode === 'wizard') {
+      $scope.watcher.$$condition_operators = [
+        {operator: '>=', name: 'Is above'},
```

Comment:
It is better to be explicit about operator names, for example:
```
operator: '>', name: 'above',
operator: '<', name: 'below',
operator: '>=', name: 'above equal'
operator: '<=', name: 'below equal'
```

Reply:
Maybe we should follow lucene queries?
- gt - greater than
- gte - greater than or equal to
- lt - less than
- lte - less than or equal to

Reply:
They may become too long, I'm trying atm. If I find them to be too long, I'll use your approach.